packages/venia-ui/lib/components/MiniCart/miniCart.js

```diff
@@ -32,9 +32,13 @@ const MiniCart = React.forwardRef((props, ref) => {
     // when the MiniCart is open.
     useScrollLock(isOpen);
 
+    const { queries, mutations } = { ...MiniCartOperations };
     const talonProps = useMiniCart({
         setIsOpen,
-        ...MiniCartOperations
+        operations: {
+            ...queries,
+            ...mutations
+        }
```

Comment:
```suggestion
```

Shouldn't need to do any of this - can just remove operations. It will be destructured from the default in the talon.

Reply:
See other comment.

Reply:
Talon does not have all needed queries, moving them next to talon will require to move a lot of imported `gql` files.